relay-general/derive/src/lib.rs

```diff
@@ -967,7 +984,7 @@ fn parse_field_attributes(
                             }
                         }
                         other => {
-                            panic!("Unexpected or bad attribute {}", other.name());
+                            panic!("Unexpected or bad attribute {:?}", other.path());
```

Comment:
Is this the ideal way to render this message?

Reply:
i think it's the only way that works for sure. to get a `Display` type you need to call functions that return fallible/optional types.